sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsLiveTests.cs

```diff
@@ -0,0 +1,555 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Errors;
+using Azure.Messaging.EventHubs.Metadata;
+using Azure.Messaging.EventHubs.Tests.Infrastructure;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of live tests for the <see cref="TrackOne.EventHubsDiagnosticSource" />
+    ///   class.
+    /// </summary>
+    ///
+    /// <remarks>
+    ///   These tests have a depenency on live Azure services and may
```

Comment:
It may be helpful to call out that these tests depend on global state and must be run serially and in isolation.   _(lest someone change the attribute)_